templates/index.html

```diff
@@ -1,130 +1,95 @@
+        <meta name="twitter:card" content="summary">
+        {% endblock meta_tags_in_head %}
+        <title>{% block title %}{{ SITENAME|striptags|e }}{% endblock title %}</title>
+        {% block head_links %}
+
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Chakra+Petch&display=swap">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
 
-    {% if articles %}
-    {% set css_class = 'span8 offset2' %}
-    {% if articles and PROJECTS and LANDING_PAGE_TITLE %}
-    {% set css_class = 'span12' %}
-    {% endif %}
-    {% from '_includes/_defaults.html' import RECENT_ARTICLES_COUNT, ARCHIVES_URL, CATEGORIES_URL with context %}
-    <div class="row-fluid">
-        <div class="{{css_class}}">
-            <header>
-            <h1 id="recent-posts">Recent Posts <a id="allposts" href="{{ SITEURL }}/{{ ARCHIVES_URL }}">all posts</a></h1>
-            </header>
-        <div class="recent-posts">
-            {% for article in articles %}
-            {% if loop.index0 < RECENT_ARTICLES_COUNT %}
-            <div itemscope class="recent-posts-article">
-                <a href="{{ SITEURL }}/{{ article.url }}">
-                    {{ article.title }}
-                    {%if article.subtitle %}
-                    <small class="subtitle">
-                        {{ article.subtitle }}
-                    </small>
-                    {% endif %}
-                </a>
-                <div class="recent-posts-posted">
-                    posted in
-                    <a href="{{ SITEURL }}/{{ CATEGORIES_URL }}#{{ article.category.slug }}-ref">{{ article.category }}</a>
-                    <div class="recent-posts-time">
-                        <time itemprop="dateCreated" datetime="{{ article.date.isoformat() }}">{{ article.locale_date }}</time>
+        {% if 'assets' in PLUGINS %}
+        {% include '_includes/minify_css.html' with context %}
+        {% else %}
+        <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/pygments.css" media="screen">
+        <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/tipuesearch/tipuesearch.css" media="screen">
+        <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/elegant.css" media="screen">
+        <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/admonition.css" media="screen">
+        <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/custom.css" media="screen">
+        {% endif %}
+        {% endblock head_links %}
+        {% include '_includes/favicon_links.html' %}
+        {% block feed_links %}
+        {% include '_includes/feeds.html' %}
+        {% endblock feed_links %}
+        {% include '_includes/analytics.html' %}
+        <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/landing_page.css" media="screen">
+        <style>
```

Comment:
This landing page-specific rule should be moved to the CSS file. This template file will probably be reused for multiple pages when we start developing the whole site.

Reply:
I'm not 100% sure at this point, but I think `index.html` is only for the landing page (others are `page`, `article`, `categories`, etc.). The reused one is `base.html`. 
That said, I don't have a problem with consolidating rules as long as it's convenient to target rules to the frontpage (e.g. if all pages have a unique `class`).